### PR TITLE
Updated GDSC-Lead Map

### PR DESCRIPTION
**Pull request by James7881**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -1046,7 +1047,22 @@ const data = [
         "linkedin": "https://linkedin.com/in/sanjaydevtech",
         "twitter": "https://twitter.com/SanjayDevTech",
         "portfolio": "https://sanjaydevtech.github.io/"
-	}
+	},
+
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:1051]:
```suggestion
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by James7881** [comment]
I submitted Jamsher Shekh details,
This code is not submitted by me

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]
@James7881 
Your photograph is missing in the repository, do add that as well.

**Comment by James7881** [comment]
photo added

**Review by abhinavsri360 — APPROVED** [review]
Looks good to me now.

**Pull request merged**